pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -207,18 +218,17 @@ public boolean orderNumberExists(String orderNumber) {
                     } else {
                         dbVals.resultValue = false;
                     }
-
-                } catch (Exception e) {
-                    throw new RuntimeException("Error getting values from db", e);
+                } catch (SQLException exception){
+                    dbVals.resultException = exception;
                 }
-            } catch (SQLException ex) {
+            } catch (Exception ex) {
```

Comment:
This catch block is unnecessary, and does not follow the pattern we typically use for DSM DB queries.